subprojects/gradle/impact/src/test/kotlin/com/avito/impact/ImpactAnalysisTest.kt

```diff
@@ -0,0 +1,439 @@
+package com.avito.impact
+
+import com.avito.test.gradle.AndroidAppModule
+import com.avito.test.gradle.AndroidLibModule
+import com.avito.test.gradle.Module
+import com.avito.test.gradle.PlatformModule
+import com.avito.test.gradle.TestProjectGenerator
+import com.avito.test.gradle.TestResult
+import com.avito.test.gradle.commit
+import com.avito.test.gradle.dir
+import com.avito.test.gradle.file
+import com.avito.test.gradle.git
+import com.avito.test.gradle.mutate
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+import java.nio.file.Path
+
+class ImpactAnalysisTest {
```

Comment:
I've revised tests for impact analysis.
Now they don't reuse a shared test project. It's easier to understand the logic.